schemas/bridging/connectionStep3Handshake.schema.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/bridging/connectionStep3Handshake.schema.json",
+  "title": "connectionStep3Handshake",
+  "type": "object",
+  "properties": {
+    "type": {
+      "const": "handshake"
+    },
+    "payload": {
+      "type": "object",
+      "properties": {
+        "implementationMetadata": {
```

Comment:
True, that's probably a result of having referenced the `ImplementationMetadata` schema which includes `app`. I'm not sure if we need the full version with app elsewhere in bridging, but I'd be loath to remove it in case others use the API schemas we add here.

To resolve we could make the app element optional. Otherwise, we might have to cut the reference and just move the rest of the `ImplementationMetadata` definition (which we do use) directly into this message (@tpina)

Reply:
fixed